editor/src/clj/editor/bundle_dialog.clj

```diff
@@ -497,7 +497,7 @@
     (let [refresh! (make-presenter-refresher this)]
       (into [(make-generic-headers "Bundle Android Application")
              (make-android-controls refresh! owner-window)]
-            (make-generic-controls refresh! variant-choices compression-choices))))
+             (make-generic-controls refresh! variant-choices compression-choices))))
```

Comment:
The indentation got messed up here for some reason. It should align with the `[` directly after `into`.
```suggestion
            (make-generic-controls refresh! variant-choices compression-choices))))
```